### DISCUSSION TITLE
[4.0] Use fixed width for icons in the backend

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#47170] [2. Cold Features]
### What needs to be fixed

<img width="127" alt="grafik" src="https://user-images.githubusercontent.com/828371/79073506-56933900-7ce7-11ea-913c-27101a59be6a.png">
<img width="279" alt="grafik" src="https://user-images.githubusercontent.com/828371/79073513-5dba4700-7ce7-11ea-9b47-a877dc38e39a.png">

The Icons in front of the titles use a different width. For a constant and harmonic look it would be great to have a fixed width for the icons in front.

**Comment by brianteeman** [comment]
all you have to do is to add the class fa-fw to all the icons. easy enough for you to do and submit a pr

**Comment by coolcat-creations** [comment]
@brianteeman I am not sure if that fixes the issue for 3rd party icons since they are not Font Awesome Icons. That's why I also posted the screen with the patch tester.

**Comment by brianteeman** [comment]
It doesnt but it is the starting point

**Comment by C-Lodder** [comment]
I don't see this as an issue personally. Icons will never be the same with. Fixed with are normally used in things like list items....where each item comes directly below each other. Seeing as these are cards and scattered around the dashboard, I don't see the need for a fixed width.

**Comment by chmst** [comment]
Agree with @C-Lodder. I think it is not possible to define a width, as icons can be changed by the user. If we have a fixed width, individual icons must be cut. Or, if they are very small, we would have a lot of free space. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28670">issues.joomla.org/tracker/joomla-cms/28670</a>.</sub>

**Comment by coolcat-creations** [comment]
Hm, I would rather see a restriction in size here otherwise extensions can break the UI completely. There is a reason why for example a apple touch icon must have a specific size otherwise the screen would look cluttered in the extensions settings overview or whereever.

**Comment by brianteeman** [comment]
If an extension looks bad it wont be long before the extension developer fixes it

**Comment by Chaosxmk** [comment]
If a fixed width is an issue, would a min-width work better? It would at least be a more flexible option.

**Comment by brianteeman** [comment]
I guess nothing is going to happen here now and this should be closed. Not my call though
